test/alterschema/alterschema_lint_2019_09_test.cc

```diff
@@ -2295,6 +2295,68 @@ TEST(AlterSchema_lint_2019_09, property_names_default_1) {
   EXPECT_EQ(document, expected);
 }
 
+TEST(AlterSchema_lint_2019_09, definitions_to_defs_1) {
+  sourcemeta::core::JSON document = sourcemeta::core::parse_json(R"JSON({
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "#/definitions/uuid"
+      }
+    },
+    "definitions": {
+      "uuid": {
+        "type": "string",
+        "format": "uuid"
+      }
+    }
+  })JSON");
+
+  LINT_AND_FIX_FOR_READABILITY(document);
+
+  const sourcemeta::core::JSON expected = sourcemeta::core::parse_json(R"JSON({
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "#/$defs/uuid"
```

Comment:
Can you try a more complicated pointer? Like a schema that has a nested schema that defines `$defs` and then it is referenced from the top as `#/foo/bar/$defs/baz` or something?

Reply:
Also with a nested schema resource maybe. Like

```json
{
    "$ref": "nested#/definitions/foo",
    ...
    "$defs": {
      "nested": {
        "$id": "nested",
        ...
        "definitions": {
          "foo": {
            "type": "string"
          }
        }
      }
    }  
}
```